app/views/free_tasks/show.html.erb

```diff
@@ -8,12 +8,20 @@
 
     <%= authenticated_only do %>
       <% if @incomplete_solution %>
-        Предай решение: <code class="solution-token">vim-fmi free <%= @incomplete_solution.token %></code>
 
+        Предай решение: <code class="solution-token" ><%= Clipboard.copy('vim-fmi put ' + @incomplete_solution.token) %></code>
```

Comment:
This copy call will copy it on the server, so we don't need it.

Reply:
My idea was to copy it automatically when requesting a new token